benchmarks/verify.mjs

```diff
@@ -9,7 +9,7 @@ async function runSuites() {
     compareVerifying(tokens[algorightm], algorightm, publicKeys[algorightm])
     return
   } else {
-    for (const algorightm of ['HS512', 'ES512', 'RS512', 'PS512', 'EdDSA']) {
+    for (const algorightm of ['HS256', 'HS512', 'ES512', 'RS512', 'PS512', 'EdDSA']) {
```

Comment:
the same things said for `sign.mjs` applies here as well

Reply:
Fixed! Let me know if you'd like to me to run the benchmarks and update the readme too